dev-haskell/base16-bytestring/base16-bytestring-1.0.1.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="RFC 4648-compliant Base16 encodings for ByteStrings"
+HOMEPAGE="https://github.com/haskell/base16-bytestring"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+# mask due to large number of packages that would break
+# KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
```

Comment:
We usually drop KEYWORDS only for ghc-related masks to ease rekeywording on arch-by-arch basis. For version-specific masks I suggest using normal `package.mask` with comment detail what should be done to unmask it.

Reply:
noted, both of these have been fixed.